kubeflow/jupyter/sync-notebook.jsonnet

```diff
@@ -115,7 +116,80 @@ function(request) {
       },
     },
   ],
-  children: children,
+  local groupByResource(resources) = {
```

Comment:
Since we're using this to build the children in several places now, we may want to move these routines into util

Reply:
Moved to util